CHANGELOG.md

```diff
@@ -1,3 +1,65 @@
+# 9.0.0-next.0 "cardboard-cpu" (2019-10-03)
+
+### cdk
+
+_Highlights_:
+* New infrastructure for creatinv component test harnesses is available in `@angular/cdk/testing`.
+  Documentation for this infrastructure and for specific Angular Material harnesses is in-progress.
+
+|            |                       |
+| ---------- | --------------------- |
+| bug fix |  **drag-drop:** dragging styles not reset once dragging is… ([#17150](https://github.com/angular/components/issues/17150)) ([e62e6bd](https://github.com/angular/components/commit/e62e6bd)), closes [#17139](https://github.com/angular/components/issues/17139) |
```

Comment:
Include last part: "dragging styles not reset once dragging is completed

Reply:
Interesting. not sure why it is cut off like that. Might be good always including the full commit title?